metadata_backend/api/handlers.py

```diff
@@ -171,18 +188,42 @@ async def delete_object(self, req: Request) -> Response:
         :returns: JSON response containing accessionId for submitted object
         """
         schema_type = req.match_info['schema']
-        if schema_type not in schema_types.keys():
-            reason = f"Specified schema {schema_type} was not found."
-            LOG.error(reason)
-            raise web.HTTPNotFound(reason=reason)
+        self._check_schema_exists(schema_type)
+        collection = (f"draft-{schema_type}" if req.path.startswith("/drafts")
+                      else schema_type)
+
         accession_id = req.match_info['accessionId']
         db_client = req.app['db_client']
-        await Operator(db_client).delete_metadata_object(schema_type,
+        await Operator(db_client).delete_metadata_object(collection,
                                                          accession_id)
-        LOG.info(f"DELETE object with accesssion ID {accession_id}"
-                 f"in schema {schema_type} was successful.")
+        LOG.info(f"DELETE object with accesssion ID {accession_id} "
+                 f"in schema {collection} was successful.")
         return web.Response(status=204)
 
+    async def put_object(self, req: Request) -> Response:
+        """Save metadata object to database.
+
+        :param req: PUT request
+        :returns: JSON response containing accessionId for submitted object
+        """
+        schema_type = req.match_info['schema']
+        accession_id = req.match_info['accessionId']
+        self._check_schema_exists(schema_type)
+        collection = (f"draft-{schema_type}" if req.path.startswith("/drafts")
+                      else schema_type)
+
+        db_client = req.app['db_client']
+        content = await req.json()
+        operator = Operator(db_client)
+        await operator.replace_metadata_object(collection,
+                                               accession_id,
+                                               content)
+        body = json.dumps({"accessionId": accession_id})
+        LOG.info(f"PUT object with accesssion ID {accession_id} "
+                 f"in schema {collection} was successful.")
+        return web.Response(body=body, status=201,
+                            content_type="application/json")
+
```

Comment:
I suppose this partially fixes issue #21 as well

Reply:
I can add a patch endpoint as well, the `update` is not that difficult but maybe different PR this one is already pretty big

Reply:
I will assign that to myself so I don't forget and will add a follow-up PR after this one is merged